src/ui/Navbar.fs

```diff
@@ -4,67 +4,125 @@ open Feliz
 open Feliz.Router
 open type Store.UI.NextUI
 open type Feliz.Html
+open Fable.Core.JsInterop
 
 [<ReactComponent>]
 let DefaultNavbar () =
+    let (isMenuOpen, setIsMenuOpen) = React.useState false
+
+    let menuItems = [ "Shop"; "About Us"; "Contact" ]
+
     Navbar [
-        prop.style [ style.backgroundColor "#ffffff" ]
+        prop.custom ("onMenuOpenChange", setIsMenuOpen)
+
+        prop.style [
+            (if isMenuOpen then "#F7F4F0" else "rgba(255,255,255,0)") |> style.backgroundColor
+            style.fontFamily "Figtree"
+            style.fontWeight 400
+            style.float'.left
+            length.px 44 |> style.minHeight
+            (if isMenuOpen then 0.0 else 0.2) |> style.transitionDelaySeconds
+        ]
 
         prop.children [
             NavbarBrand [
                 prop.children [
-                    p [
-                        prop.text "ZORI"
-                        prop.className [ "font-bold"; "text-inherit" ]
+                    img [
+                        prop.src <| import "default" "./assets/Logo.svg"
                         prop.onClick (fun _ -> Router.navigatePath "/")
-                        prop.style [ style.cursor "pointer" ]
+                        prop.style [
+                            length.px 26 |> style.height
+                            length.px 26 |> style.minHeight
+                            style.cursor "pointer"
+                            (if isMenuOpen then 0 else 100) |> style.filter.invert
```

Comment:
This probably needs a transition. It disappears for a sec when I toggle the menu until the wipe catches up.

Reply:
Added transition.